docs/run/run-primitives-batch.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Run primitives in a batch
+description: How to run primitives in batch mode. 
+
+---
+
+# Run primitives in a batch
+
+Batch mode is ideal for running experiments closely together to avoid device drifts, that is, to maintain device characterization.  All jobs must be provided at the outset. If you want to submit iterative jobs, use [sessions](sessions) instead.
```

Comment:
Is the note about drift necessarily true? If I have a large experiment that takes several hours, I'll still have drift

Reply:
I'm not sure - it came from the existing Batch content on the Sessions page.  @jyu00 ?

Reply:
@pandasa123 you're right that the statement isn't always true. I think the intent here is to describe when you want to use Batch vs single jobs vs session. Some reasons to use batch would be (1) you want to avoid delays between jobs (e.g. to avoid drift) and (2) to speed things up using classical parallelism. 

tbh I think the opening paragraph can use a bit of rewording. It starts by saying it's ideal for avoiding drift, which can be true but not always. Then it lists 2 bullets points, which I think are meant to demonstrate the benefits, but "Suitable for batching many jobs together" isn't exactly a benefit. To me the biggest selling point of Batch is the speed up you gain by having multiple jobs run in parallel.